Client/src/components/Shared/Navbar/Navbar.jsx

```
@@ -1,135 +1,158 @@
-              </button>
+                <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-full shadow-md"></span>
+                <span className="absolute inset-0 w-full h-full bg-white/15 rounded-full blur-[2px]"></span>
+                <span className="absolute inset-0 w-full h-full bg-blue-600 rounded-full transform transition-transform group-hover:scale-105"></span>
+                <span className="relative">Login</span>
+              </Link>
             )}
             
-            {/* Mobile Menu Toggle */}
-            <Navbar.Toggle className="ml-2" />
+            {/* Mobile menu button */}
+            <button className="md:hidden text-gray-300 hover:text-white focus:outline-none">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)